### DISCUSSION TITLE
ScrollPanel - how to keep the scrollbar always visible?

### DISCUSSION THREAD

**Discussion by VilleH-RockOn** [primefaces/community#875] [PrimeReact] [answered]
Hi, in the documentation page https://primereact.org/scrollpanel/ the custom example somehow keeps the scrollpanel always visible even without mouseover.

How is this done? The codesandbox example https://codesandbox.io/s/wmjh5f is broken and missing the ScrollPanelDemo.css file so I cannot view it from there.

**Comment by melloware** [comment, marked as answer]
Here is a working reproducer including that CSS: https://stackblitz.com/edit/56ap19?file=src%2FScrollPanelDemo.css

**Reply by VilleH-RockOn** [reply]
Thank you :)
